_docs/customer-stories/fec.md

```diff
@@ -17,10 +17,10 @@ The Federal Election Commission (FEC) recently worked with 18F to redesign its w
 
 ## $1.2 million annual savings
 
-Political campaigns and committees across the country periodically submit financial reports to the FEC which are then disclosed to the public on FEC.gov. Around elections and filing deadlines, the website experiences significant spikes in traffic far in excess of normal demand. To ensure the old website was responsive and available enough, the FEC had to predict future spikes and build for them. To that end, FEC spent $1.4 million annually on the data center supporting the old FEC.gov. This included servers and their maintenance, databases, and various software licenses. According to the FEC’s Deputy CIO Wei Luo, maintaining all of this was expensive and “could take up to 3 months.” Since beginning their migration to cloud services, including cloud.gov, Luo says they plan to retire all 14 servers by the end of the fiscal year.
+Political campaigns and committees across the country periodically submit financial reports to the FEC which are then disclosed to the public on FEC.gov. Around elections and filing deadlines, the website experiences significant spikes in traffic far in excess of normal demand. To ensure the old website was responsive and available enough, the FEC had to predict future spikes and build for them. To that end, FEC spent $1.4 million annually on the data center supporting the old FEC.gov. This included servers and their maintenance, databases, and various software licenses. According to the FEC’s Deputy CIO Wei Luo, maintaining all of this was expensive and “could take up to 3 months” to purchase and deploy new servers. Since beginning their migration to cloud services, including cloud.gov, the FEC has been able to retire all 14 servers.
```

Comment:
Changes:
>maintaining all of this was expensive and “could take up to 3 months” **to purchase and deploy new servers.**

>Since beginning their migration to cloud services, including cloud.gov, the FEC **has been able to retire all 14 servers.**
(References Waltham data center)